liboptv/include/sortgrid.h

```diff
@@ -0,0 +1,39 @@
+/* Utilities for handling sortgrid */
+
+#ifndef SORTGRID_H
+#define SORTGRID_H
+
+#include "calibration.h"
+#include "parameters.h"
+#include "tracking_frame_buf.h"
+#include "imgcoord.h"
+#include "multimed.h"   
+#include "trafo.h"
+#include "vec_utils.h"
+#include <stdio.h>
+
+typedef struct
+{
+  int pnr;
+  vec3d pos;
+} coord_3d;
```

Comment:
As far as I can see, pnr isn't really necessary. It's only used for numbering pix[], but since the points are read and numbered in the order of the file, we can just use the point index for that. I suggest to drop (again) the coord_3d structure and simply use vec3d[] as the known-points data structure. That's what I did in my bindings into PyPTV and it works great.